src/web/js/error-ui.js

```diff
@@ -137,7 +176,7 @@
               .append($("<p>").text("Error: "))
               .append(exnstringContainer)
               .append($("<p>"))
-              .append(outputUI.renderStackTrace(runtime, editors, srcloc, e));
+              .append(outputUI.renderStackTrace(runtime, documents, srcloc, e.pyretStack));
```

Comment:
Was this just a bug before this diff, or did the signature of `renderStackTrace` change?